high-level-tests/e2e/cypress/fixtures/users.json

```
@@ -65,5 +65,14 @@
     "username": "user.shouldChangePassword1234",
     "password": "$2b$05$Ryd6vVDoubvN9MQmSGmfY.tscxdmWhu1g8NOrHI4dveemziMof6Qm",
     "shouldChangePassword": true
+  },
+  {
+    "id": 9,
+    "firstName": "Joffrey",
+    "lastName": "Baratheon",
+    "email": "userWhoMustValidateTheLatestTermsOfService@example.net",
```

```suggestion
    "email": "userWhoMustValidateTheLastTermsOfService@example.net",
```
Pour garder la cohérence avec le champ de BDD, les use-case (cf remarque de @yahyayouness sur accept/validate)
Mais stricto sensu, [vous avez raison](http://masterkeyenglish.com/last-vs-latest/)